sway-ir/Cargo.toml

```diff
@@ -14,6 +14,7 @@ downcast-rs = "1.2.0"
 filecheck = "0.5"
 generational-arena = "0.2"
 itertools = "0.10.3"
+once_cell = "1.18.0"
```

Comment:
This isn't actually used.
```suggestion
```

Reply:
Actually it is: `use once_cell::sync::Lazy;` Used only once :smile: in unit tests to save a line of code in every test setup.